awx/main/tests/docs/conftest.py

```diff
@@ -0,0 +1,13 @@
+from awx.main.tests.functional.conftest import *  # noqa
+
+
+def pytest_addoption(parser):
```

Comment:
This allows you to invoke the doc build test with `py.test awx/main/tests/docs --release=3.3.0`.